### DISCUSSION TITLE
How to get cluster wise cell numbers for each replicate and do the statistical tests?

### DISCUSSION THREAD

**Discussion by sghoshuc** [satijalab/seurat#8988] [Q&A]
Hi Satija lab, We recently conducted scRNA-seq analysis with samples from two different experimental groups and a control group. Each group consists of 8 replicates, and the samples were obtained from male and female animals. We've noticed differences in cell numbers among clusters between the groups. Is there a method to retrieve the cell numbers for each replicate? Additionally, would it be appropriate to use a t-test or ANOVA to assess whether the differences in cell numbers between groups are statistically significant? Furthermore, is it common to observe differences in cell numbers between replicates, and if not, how should I address this issue? Thank you for your assistance.

**Comment by wudustan** [comment]
> Is there a method to retrieve the cell numbers for each replicate?

If you have the replicate as a column in the cell meta-data (eg. `seurat.object$replicate`) then you can do `table(seurat.object$replicate)`

> Additionally, would it be appropriate to use a t-test or ANOVA to assess whether the differences in cell numbers between groups are statistically significant?

You could try something like [scCODA](https://www.nature.com/articles/s41467-021-27150-6) if you wanted to rigorously test this but also you could try the more simple statistical tests you listed and see what kind of result you get and if it makes sense with any intrepretation?

> is it common to observe differences in cell numbers between replicates

This depends entirely on your experiment. There will always be sampling issues when it comes to collecting tissue and tissue preparation. It also depends on the cell type - eg. immune, cycling, is it a tumour cell and the tumours are different size/composition, etc etc. Are you simply getting a lower number of total cells between replicates or are cell numbers roughly the same and only certain clusters have different proportions.
